rdesktop.c

```diff
@@ -697,6 +700,16 @@ main(int argc, char *argv[])
 					p++;
 				}
 
+				if (*p == '@')
+				{
+					g_dpi = strtol(p + 1, &p, 10);
```

Comment:
It would be, but I cannot write it. My box gets the wrong value for all programs. :(